tests/slurm-tests/qwen3_4b_ray_executor/check_results.py

```
@@ -0,0 +1,97 @@
+    )
+
+    status_file = os.path.join(marker_dir, "status.json")
+    soft_assert(
+        os.path.isfile(status_file),
+        f"expected status.json at {status_file}",
+    )
+
+    if os.path.isfile(status_file):
+        try:
+            status = load_json(status_file)
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)